domain/lib/src/model/biometric_authentication/authentication_biometric_state.dart

```diff
@@ -29,4 +29,4 @@
 //  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
 //  the Additional Terms applicable to LinShare software.
 
-enum AuthenticationBiometricState { authenticated, unAuthenticated, unEnrolled }
\ No newline at end of file
+enum AuthenticationBiometricState { authenticated, unAuthenticated, unEnrolled, locked, rejected, none }
```

Comment:
Btw unauthenticated and unenrolled are single words so I don't think camel case is needed

Reply:
True unauthenticated is a single word, but unEnrolled is a compound word

Reply:
By single words I mean it's written unrolled and not un-rolled or un rolled so it's not a camel case :(

Reply:
unEnrolled not unrolled

Reply:
yes yes it's the same logic 😭 Well I don't mind it's just a detail